pkg/api/archivist/archivist.proto

```diff
@@ -23,20 +23,21 @@ import "google/protobuf/empty.proto";
 message GetBySubjectDigestRequest {
   string algorithm = 1;
   string value = 2;
+  optional string collectionName = 3;
```

Comment:
Optional is a noop in proto3. All fields are considered optional in the line protocol and code generation. Our application should enforce logic regarding required/optional instead.